.semaphore/semaphore.yml

```diff
@@ -13,10 +13,13 @@ blocks:
     task:
       prologue:
         commands:
-          - sem-version php 7.2.10 # Be explicit, even when using the default (in case the default changes)
+          - sem-version php $PHP_VERSION
           - checkout
       jobs:
       - name: Composer install
+        matrix:
+          - env_var: PHP_VERSION
+            values: ["7.2.10", "7.3.0"] # Be explicit, even when using the default (in case the default changes)

```

Comment:
Is the comment here correct now?

Reply:
@maxakropolis Good point, and you got me thinking that perhaps it should say:

> ["7.2.10", "7.3.0"] # Specify patch versions, not major.minor aliases (to avoid surprises)

...but that actually seems a bad idea, and that we should be doing the exact opposite:

> ["7.2", "7.3"] # These are aliases that resolve to whichever patch version is available

Reply:
@maxakropolis It seems like there are pros and cons to both approaches, so something to discuss and investigate separately.